boxoffice/models/item.py

```diff
@@ -105,3 +184,38 @@ class Price(BaseScopedNameMixin, db.Model):
 
     amount = db.Column(db.Numeric, default=Decimal(0), nullable=False)
     currency = db.Column(db.Unicode(3), nullable=False, default=u'INR')
+
+    __roles__ = {
+        'price_owner': {
+            'write': {},
+            'read': {'id', 'item_id', 'json_start_at', 'json_end_at', 'amount', 'currency', 'discount_policy_title', 'tense'}
+        }
+    }
+
+    def roles_for(self, actor=None, anchors=()):
+        roles = super(Price, self).roles_for(actor, anchors)
+        if self.item.item_collection.organization.userid in actor.organizations_owned_ids():
+            roles.add('price_owner')
+        return roles
+
+    @property
+    def discount_policy_title(self):
+        return self.discount_policy.title if self.discount_policy else None
+
+    @property
+    def json_start_at(self):
+        return localize_timezone(self.start_at).isoformat()
+
+    @property
+    def json_end_at(self):
+        return localize_timezone(self.end_at).isoformat()
+
+    @property
+    def tense(self):
```

Comment:
Not a property since the result changes with time even if the data does not change.